ee/src/main/java/org/jboss/as/ee/subsystem/ContextServiceResourceDefinition.java

```diff
@@ -59,7 +67,10 @@
     public static final ContextServiceResourceDefinition INSTANCE = new ContextServiceResourceDefinition();
 
     private ContextServiceResourceDefinition() {
-        super(PathElement.pathElement(EESubsystemModel.CONTEXT_SERVICE), EeExtension.getResourceDescriptionResolver(EESubsystemModel.CONTEXT_SERVICE), ContextServiceAdd.INSTANCE, ContextServiceRemove.INSTANCE);
+        super(new SimpleResourceDefinition.Parameters(PathElement.pathElement(EESubsystemModel.CONTEXT_SERVICE), EeExtension.getResourceDescriptionResolver(EESubsystemModel.CONTEXT_SERVICE))
+                .setAddHandler(ContextServiceAdd.INSTANCE)
+                .setRemoveHandler(ContextServiceRemove.INSTANCE)
```

Comment:
.setRemoveHandler(new ServiceRemoveStepHandler(ContextServiceAdd.INSTANCE))

Reply:
this change will tag such line of code as deprecated, cause
it will see it as "public ServiceRemoveStepHandler(final AbstractAddStepHandler addOperation, final RuntimeCapability ... unavailableCapabilities)", which is deprecated...
Instead of "protected ServiceRemoveStepHandler(final AbstractAddStepHandler addOperation)" when you extend it

Reply:
This will be resolved in the next wildfly-core upgrade.